IFC4x3/Properties/d/DiscontinuityWaterSeapage_37bDGHt1D5IeRZslC_HS3J/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DiscontinuityWaterSeapage_37bDGHt1D5IeRZslC_HS3J" Name="DiscontinuityWaterSeapage" UniqueId="c794d411-dc13-454a-86e3-daf33e45c0d3" PrimaryDataType="IfcLabel" />
```

Comment:
DiscontinuityWaterSeapage typo? DiscontinuityWaterSeepage?

Also, `IfcLabel`?

Reply:
> DiscontinuityWaterSeapage typo? DiscontinuityWaterSeepage?
> 
> Also, `IfcLabel`?

Should be "DiscontinuityWaterSeepage". IfcLabel is correct.